webapp/templates/proxy-create.xhtml

```diff
@@ -0,0 +1,303 @@
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <div class="col-sm-6">
+                <aside class="well">
+                    <p>PURLs are used to curate the URL resolution process, thus solving
+                    the problem of transitory URIs in location-based URI schemes like HTTP.</p>
+                    <p>Partial PURLs are a special class of PURL that must end with a '/' and 
+                    allow a hierarchy of URLs to be mapped to another resource. The request pattern
+                    may include regular expressions in the path subsequent to the PURL name.</p>
+                    <p>Other PURLs cannot end with a '/' and the request pattern only applies to the query string.</p>
+                </aside>
+            </div>
+        </div>        
+                
+        <div id="ruleOptions" style="display:none">
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/><!-- special case for viewing when no GET rule -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+        <input id="cacheProperty" type="hidden" property="calli:cacheControl" onchange="calli.updateProperty(event, 'calli:cacheControl')"/>
+                
+        <div class="row">
+            <div id="rules" class="col-sm-12">
+                <label>Handlers</label>
+
+                <div class="panel panel-default rule" data-method="GET">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">GET Handler
+                            <span class="pull-right">
+                                <a href="#getHandler" data-toggle="collapse" class="glyphicon glyphicon-minus"></a>
+                            </span>
+                        </h3>
+                    </div>
+                    <div class="panel-body collapse in" id="getHandler">
+                        <fieldset class="row">
+                            <div class="form-group col-sm-2">
+                                <label for="getActionType">Action</label>
+                                <div>
+                                    <select id="getActionType" class="form-control actionType">
+                                        <option value="calli:copy">Copy (200)</option>
```

Comment:
Don't include the (200) here, since the status code is also copied from the target response (and might not be 200). Instead use the following text: "Perform GET", "Perform POST", "Perform PUT", "Perform PATCH", and "Perform DELETE"